docs/advanced/command-line/files/data-context.md

```diff
@@ -0,0 +1,20 @@
+# Data Context
+
+This file describes resources in the data context that should be included in the data wrangling
+scenario. 
+
+## Format of the file
+We have provided an example of how the contents of the file should be structured in **datacontext.csv** in the Manual Books example.
+
+The first 8 columns in this file are identical to the ones describing the [sources](../sources/): uri, tables, description, data, extensions, separator, recursive, overwrite. 
+
+The following 3 columns, however, columns 9, 10, and 11, should be as follows:
```

Comment:
You can just simplify this to just: However, columns 9, 10, and 11 should be as follows.

Reply:
Done!